src/integrationTest/java/com/mongodb/hibernate/BasicCrudIntegrationTests.java

```diff
@@ -199,23 +199,20 @@ void testDynamicUpdate() {
     }
 
     @Nested
-    class LoadByIdTests {
+    class LoadByPrimaryKeyTests {
 
         @Test
-        void testLoad() {
+        void testGetById() {
```

Comment:
The class uses the "PrimaryKey" wording, while the test uses the "Id` wording. The latter seems more appropriate for us, but I'll be OK with either. However, can we make the wording consistent?

Reply:
I adopted the naming. But as you see, even this `loadByPrimaryKeyTests` ended up with multiple testing cases suggested by you; so creating a `@Nested` in anticipation of such grouping need is not a bad thing from my understanding.